### PR TITLE
Release 27.8

### PR DESCRIPTION
**Pull request by lucasmoura**

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,17 @@
+ubuntu-advantage-tools (27.8~22.04.1) jammy; urgency=medium
+
+  * New upstream release 27.8 (LP: #1969125)
+    - entitlements: apply overrides from the contract response
+    - fips:
+      + unhold fips package when enabling fips-updates
+      + make fips service incompatible with fips-updates
+      + unhold more packages when enabling fips
+    - lib: fix upgrade script for unsupported releases
```
Comment (orndorffgrant) [inline debian/changelog:9]: This fixes a LP bug 1968067

File: debian/changelog [hunk 2]
```diff
@@ -1,3 +1,17 @@
...
+    - realtime: add suport for realtime kernel beta service on Jammy
```
Comment (orndorffgrant) [inline debian/changelog:10]: small typo
```suggestion
    - realtime: add support for realtime kernel beta service on Jammy
```

File: debian/changelog [hunk 3]
```diff
@@ -1,3 +1,17 @@
+ubuntu-advantage-tools (27.8~22.04.1) jammy; urgency=medium
+
+  * New upstream release 27.8 (LP: #1969125)
+    - entitlements: apply overrides from the contract response
+    - fips:
+      + unhold fips package when enabling fips-updates
+      + make fips service incompatible with fips-updates
```
Comment (orndorffgrant) [inline debian/changelog:7]: This makes sense to me, but may be confusing to others. Consider:
```suggestion
      + Automatically disable fips service before enabling fips-updates
```

File: debian/changelog [hunk 4]
```diff
@@ -1,3 +1,17 @@
+ubuntu-advantage-tools (27.8~22.04.1) jammy; urgency=medium
+
+  * New upstream release 27.8 (LP: #1969125)
+    - entitlements: apply overrides from the contract response
+    - fips:
+      + unhold fips package when enabling fips-updates
```
Comment (orndorffgrant) [inline debian/changelog:6]: another small typo
```suggestion
      + unhold fips packages when enabling fips-updates
```

File: debian/changelog (hunk 3 above)
Comment (lucasmoura) [reply debian/changelog:7]: Good point, updating

**Comment by lucasmoura** [comment]
@orndorffgrant updated

**Review by orndorffgrant — APPROVED** [review]

**Pull request merged**